### PR TITLE
Validators list

### LINKED ISSUE #1437: Validators list
**Validators page** should provide detailed information about the validators of the PoS chains. This page includes:

- summary statistics on validators
- a full list of validators with detailed information about each of them. 

Requirements: https://www.notion.so/blockscout/Validators-Page-999f17d3801e41d1aabf14cf41401e8d?pvs=4

UI: TBD, under review

Backend task: TBD

### PR DESCRIPTION
**Pull request by tom2drum**
## Description and Related Issue(s)

Resolves #1437

### Proposed Changes
Added variable `NEXT_PUBLIC_VALIDATORS_CHAIN_TYPE` for enabling validators page

## Checklist for PR author
- [x] I have tested these changes locally.
- [x] I added tests to cover any new functionality, following this [guide](./CONTRIBUTING.md#writing--running-tests)
- [ ] Whenever I fix a bug, I include a regression test to ensure that the bug does not reappear silently.
- [x] If I have added, changed, renamed, or removed an environment variable, I have updated the list of environment variables in the [documentation](ENVS.md) and  made the necessary changes to the validator script according to the [guide](./CONTRIBUTING.md#adding-new-env-variable)

### CODE REVIEW

File: ui/pages/Validators.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import Sort from 'ui/shared/sort/Sort';
+import { SORT_OPTIONS } from 'ui/validators/utils';
+import ValidatorsCounters from 'ui/validators/ValidatorsCounters';
+import ValidatorsFilter from 'ui/validators/ValidatorsFilter';
+import ValidatorsList from 'ui/validators/ValidatorsList';
+import ValidatorsTable from 'ui/validators/ValidatorsTable';
+
+const Validators = () => {
+  const router = useRouter();
+  // const [ searchTerm, setSearchTerm ] = React.useState(getQueryParamString(router.query.address_hash) || undefined);
```
Comment (isstuev) [inline ui/pages/Validators.tsx:32]: will this be implemented later?

File: ui/validators/ValidatorsListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const ValidatorsListItem = ({ data, isLoading }: Props) => {
+
+  return (
+    <ListItemMobileGrid.Container>
+
+      <ListItemMobileGrid.Label isLoading={ isLoading }>Address</ListItemMobileGrid.Label>
+      <ListItemMobileGrid.Value>
+        <AddressEntity
+          isLoading={ isLoading }
+          address={ data.address }
```
Comment (isstuev) [inline ui/validators/ValidatorsListItem.tsx:24]: I suggest using constant truncation here, at least for mobile view

**Review by isstuev — APPROVED** [review]

File: deploy/values/review/values.yaml.gotmpl [hunk 3]
```diff
@@ -51,7 +51,7 @@ frontend:
     NEXT_PUBLIC_FEATURED_NETWORKS: https://raw.githubusercontent.com/blockscout/frontend-configs/dev/configs/featured-networks/eth-goerli.json
     NEXT_PUBLIC_NETWORK_LOGO: https://raw.githubusercontent.com/blockscout/frontend-configs/main/configs/network-logos/goerli.svg
     NEXT_PUBLIC_NETWORK_ICON: https://raw.githubusercontent.com/blockscout/frontend-configs/main/configs/network-icons/goerli.svg
-    NEXT_PUBLIC_API_HOST: eth-goerli.blockscout.com
+    NEXT_PUBLIC_API_HOST: stability-testnet.blockscout.com
```
[the remark this answers is not in the document]
Comment (tom2drum) [reply deploy/values/review/values.yaml.gotmpl:54]: only for demo needs

File: ui/pages/Validators.tsx (hunk 1 above)
Comment (tom2drum) [reply ui/pages/Validators.tsx:32]: yes, I hope so
this filter should be added in API in one of the next releases

**Pull request merged**